SpatialGDK/Source/SpatialGDK/Public/Utils/ActorGroupManager.h

```diff
@@ -0,0 +1,85 @@
+#pragma once
+
+#include "CoreMinimal.h"
+#include "GameFramework/Actor.h"
+#include "SpatialConstants.h"
+
+#include "ActorGroupManager.generated.h"
+
+USTRUCT()
+struct FActorClassSet
+{
+	GENERATED_BODY()
+
+	UPROPERTY(EditAnywhere)
+	TSet<TSoftClassPtr<AActor>> ActorClasses;
+
+	FActorClassSet()
+	{
+		ActorClasses = {};
+	}
+
+	FActorClassSet(TSet<TSoftClassPtr<AActor>> Classes)
+	{
+		ActorClasses = Classes;
+	}
+};
+
+USTRUCT()
+struct FWorkerAssociation
```

Comment:
Why not just replace this struct with a `using FWorkerAssociation = TMap<FName, FName>`